node/node_test.go

```diff
@@ -528,9 +537,12 @@ func TestNode_GetAllIssuedESDTs(t *testing.T) {
 
 				return ch, nil
 			},
+			RootCalled: func() ([]byte, error) {
```

Comment:
if `TrieStub` would have a nil check for the handler functions, then you wouldn't need this nil func implementation